README.md

```diff
@@ -1,56 +1,85 @@
-# <div align="left"><img src="https://rapids.ai/assets/images/rapids_logo.png" width="90px"/>&nbsp;Cyber Log Accelerators (CLX)</div>
+# <div align="left"><img src="https://rapids.ai/assets/images/rapids_logo.png" width="90px"/>&nbsp;&nbsp;Cyber Log Accelerators (CLX)</div>
 
 **NOTE:** For the latest stable [README.md](https://github.com/rapidsai/clx/blob/master/README.md) ensure you are on the `master` branch.
 
 CLX ("clicks") provides a collection of [RAPIDS](https://rapids.ai/) examples for security analysts, data scientists, and engineers to quickly get started applying RAPIDS and GPU acceleration to real-world cybersecurity use cases.
 
 The goal of CLX is to:
 
-1. Provide SIEM integration with GPU compute environments via RAPIDS and effectively extend the SIEM environment,
-1. Make available pre-built use cases that demonstrate CLX and RAPIDS functionality that are ready to use in a Security Operations Center (SOC), 
-1. Allow cyber data scientists and SecOps teams to generate workflows, using cyber-specific GPU-accelerated primitives and methods, that let them interact with code using security language, and
-1. Accelerate log parsing in a flexible, non-regex method.
+1. Allow cyber data scientists and SecOps teams to generate workflows, using cyber-specific GPU-accelerated primitives and methods, that let them interact with code using security language,
+1. Make available pre-built use cases that demonstrate CLX and RAPIDS functionality that are ready to use in a Security Operations Center (SOC),
+1. Accelerate log parsing in a flexible, non-regex method. and
+1. Provide SIEM integration with GPU compute environments via RAPIDS and effectively extend the SIEM environment.
 
-
-## Getting Started
-
-CLX is targeted towards cybersecurity data scientists, senior security analysts, threat hunters, and forensic investigators. Data scientists can use CLX in traditional Python files and Jupyter notebooks. CLX also includes structure in the form of a workflow. A workflow is a series of data transformations performed on a [GPU dataframe](https://github.com/rapidsai/cudf) that contains raw cyber data, with the goal of surfacing meaningful cyber analytical output. Multiple I/O methods are available, including Kafka and on-disk file stores.
-
-Example flow workflow reading and writing to file:
+## Getting Started with Python and Notebooks
```

Comment:
@brhodes10 Can you review this new getting started section?

Reply:
I like how `cuML` and a few other teams have a "bite-sized" code snippet highlighting the library itself and link to more fleshed out notebooks/use-cases following afterwards. Do you think this section should be more along the lines of "Getting Started with CLX" and give a more condensed code-snippet highlighting features in the CLX library?

Reply:
This is a good idea. Going to move it into a new issue ([issue 112](https://github.com/rapidsai/clx/issues/112)).